easybuild/easyconfigs/g/GROMACS/GROMACS-2018.2-foss-2018b.eb

```diff
@@ -0,0 +1,39 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```

Comment:
@akesandgren `hpcugent` should be `easybuilders` (the tests should trip over this...)